lib/worker.js

```diff
@@ -84,30 +80,78 @@ async function getLocalLinter(basedir) {
             linter = require('loophole').allowUnsafeNewFunction(() => require(linterPath).Linter);
           }
         } else {
-          linter = tslintDef;
+          return resolve();
```

Comment:
:bug: You should just need to call `resolve()`, I'm not actually sure what `return resolve()` would do, I'm guessing that `resolve()` returns some value that makes that make sense as I've seen that elsewhere without issues, but the `return` isn't needed.

Reply:
I noticed this too. There shouldn't be I'll effect, but for consistency I agree we should remove it

Reply:
Good call. I was doing it (and I suspect others do it) in order to kick execution back up the stack immediately, hence avoiding the caching operation on line 85... but that doesn't really need to be outside of the if-block anyway, so moving that inside and then getting rid of the `else` seems like a cleaner option.